src/components/EditItemForm/EditItemDescription/EditItemDescription.jsx

```diff
@@ -32,11 +32,22 @@ const EditItemDescription = (props) => {
             <div className={styles.field}>
               <label>Category</label>
               <input
-                name="category"
                 type="text"
-                value={props.itemCategory}
+                list="categories"
+                name="category"
+                defaultValue={props.itemCategory}
                 onChange={props.handleChange}
               />
+              <datalist id="categories">
+                <option name="category" value="" disabled>
+                  Choose a Category
```

Comment:
@thomasdahlberg Can you review this? Is it supposed to be visible or did you want to use it as a placeholder? I used Chrome and it doesn't seem to be working.

Reply:
You're right. This is a little clunky. I have changed it so that it is an empty string for new items and the existing category for updated items. I removed the "Choose a Category" option because it doesn't jive with the autocomplete functionality well.